jpa/org.eclipse.persistence.jpa/src/org/eclipse/persistence/internal/jpa/metadata/accessors/objects/MetadataAsmFactory.java

```diff
@@ -286,6 +286,10 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
         @Override
         public AnnotationVisitor visitAnnotation(String desc, boolean visible) {
             boolean isJPA = false;
+            if ( desc.startsWith( "Lkotlin" ) ) {
```

Comment:
@Tomas-Kraus I would ask you to consider to add this in the final solution as it ignores Kotlin annotations. I think it's important because most of the language features in Kotlin is based on Annotations that has nothing to do with JPA and ignoring them will speed up Eclipse link metadata scans.

Reply:
Yes, I'll add it too.